src/D2L.CodeStyle.TestAnalyzers/IgnoreAttribute/IgnoreAttributeAnalyzer.cs

```diff
@@ -42,16 +42,12 @@ internal sealed class IgnoreAttributeAnalyzer : DiagnosticAnalyzer {
 
 			foreach( var attribute in root.Attributes ) {
 				if( attribute.Name.ToString().Equals( "Ignore" ) ) {
-					var parentClassDeclaration = root.AncestorsAndSelf().OfType<ClassDeclarationSyntax>().ToImmutableArray().Single();
-					foreach( var parentAttributeList in parentClassDeclaration.AttributeLists ) {
-						foreach( var parentAttribute in parentAttributeList.Attributes ) {
-							if( parentAttribute.Name.ToString().Equals( "ReflectionSerializer" ) ) {
-								return;
-							}
-						}
+					var memberGroups = context.SemanticModel.GetMemberGroup( attribute );

```

Comment:
Interesting. I'm not super familiar with `GetMemberGroup`, but it seems to operate on method/property groups only. Why did you chose this one over `GetSymbolInfo`? (not saying you have to use the other one, just curious around the choice here).